lib/Server.js

```diff
@@ -270,8 +270,10 @@ class Server {
     const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
     const hot = this.hot;
     const publicPath = hot ? (webpackURL + '/') : null;
+    const platform = this.entry.substring(this.entry.search(/\./) + 1);
```

Comment:
This pulls through from CLI with existing default. I'm not happy with the
string parsing because it's fickle and error-prone without some guards. But
it works.
On Sun, Oct 25, 2015 at 1:12 AM Philipp von Weitershausen <
notifications@github.com> wrote:

> In lib/Server.js
> <https://github.com/mjohnston/react-native-webpack-server/pull/111#discussion_r42941566>
> :
>
> > @@ -270,8 +270,10 @@ class Server {
> >      const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
> >      const hot = this.hot;
> >      const publicPath = hot ? (webpackURL + '/') : null;
> > +    const platform = this.entry.substring(this.entry.search(/\./) + 1);
>
> This feels a bit hack-ish to me. An explicit --platform ios command line
> switch would seem cleaner. And it's also how the RN packager cli works.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mjohnston/react-native-webpack-server/pull/111/files#r42941566>
> .
>